server/models/dnt-policy/dnt-policy-1.0.txt

```diff
@@ -0,0 +1,218 @@
+Do Not Track Compliance Policy 
```

Comment:
Maybe we could create a directory like `server/static`? Because it's not really a model

Reply:
:+1: